pom.xml

```diff
@@ -78,11 +78,23 @@
     </dependencies>
   </dependencyManagement>
   <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.13.2</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
This is provided by the parent pom.  It is not needed in this pom file.
```suggestion
```

Reply:
Removed.